ports/cxxgraph/portfile.cmake

```
@@ -0,0 +1,16 @@
+# Header-only library

```

AFAIU there is no point in `vcpkg_cmake_configure` setting up the debug build type.
```suggestion
set(VCPKG_BUILD_TYPE release) # Header-only library
```